.github/workflows/sdk-execution.yml

```diff
@@ -0,0 +1,74 @@
+name: KFP SDK execution tests
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/sdk-execution.yml'
+      - 'sdk/python/**'
+      - 'api/v2alpha1/**'
+
+jobs:
+  sdk-execution-tests:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Create k8s Kind Cluster
+        uses: container-tools/kind-action@v2
+        with:
+          cluster_name: kfp
+          kubectl_version: v1.29.2
+          version: v0.22.0
+          node_image: kindest/node:v1.29.2
+
+      - name: Build images
+        run: ./scripts/deploy/github/build-images.sh
+
+      - name: Deploy KFP
+        run: ./scripts/deploy/github/deploy-kfp.sh
```

Comment:
You can use the `kfp-cluster` action here. Like https://github.com/kubeflow/pipelines/blob/7d2ec3951594b74043c7c83822536a75fadb1055/.github/workflows/e2e-test.yaml#L27

Reply:
Done, updated to use this action.